src/main/java/org/elasticsearch/index/analysis/ComboAnalysisBinderProcessor.java

```diff
@@ -1,30 +0,0 @@
-/*
```

Comment:
This was mostly boilerplate, now we instead use the provider directly in the plugin class.